.github/workflows/nightly.yaml

```diff
@@ -0,0 +1,29 @@
+name: Nightly
+on:
+  schedule:
+    - cron: '30 12 * * *'
+permissions:
+  id-token: write
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cache/go-build
+            ~/go/pkg/mod
+            ~/go/bin/
+            ~/.kubebuilder/bin/k8s
+          key: ${{ runner.os }}-go-cache-${{ hashFiles('**/go.sum') }}
+      - uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: arn:aws:iam::112855948879:role/nightly
+          aws-region: us-east-1
```

Comment:
Is this your role and region? Can we do us-west-2? Should we have builds for different regions? Or is this because of ecr-public?

Reply:
It's the default region, the role is made through the infra piece.

Reply:
We don't need to have ECRs in each region, what's the reason for using us-west-2? this is already made and tested it's not super easy to change it. us-east-1 as the default region has a lot of capacity and is arguably better geographically positioned to serve the world although the outcome is most likely cached.

Reply:
just to note, us-east-1 is the only region where ecr-public is available anyways, but ecr-public replicates out the images to its CDN.